src/swap/defi/thorchain.ts

```diff
@@ -237,15 +245,19 @@ const asQuoteSwap = asObject({
   memo: asString, // "=:ETH.WBTC-0X2260FAC5E5542A773AA44FBCFEDF7C193BC2C599:0x04c5998ded94f89263370444ce64a99b7dbc9f46:0/10/0",
   // outbound_delay_blocks: asNumber, // 114,
   // outbound_delay_seconds: asNumber, // 684,
-  // recommended_min_amount_in: asString, // "1440032",
+  recommended_min_amount_in: asString, // "1440032",
   router: asOptional(asString), // "0xD37BbE5744D730a1d98d8DC97c42F0Ca46aD7146",
   // slippage_bps: asNumber, // 92,
   // streaming_slippage_bps: asNumber, // 5,
   streaming_swap_blocks: asNumber, // 170,
   total_swap_seconds: asOptional(asNumber) // 1020,
 })
 
+const asQuoteSwapFull = asEither(asQuoteError, asQuoteSwap)
```

Comment:
Why don't we use this in `getQuote`?

Reply:
Good point. Really it's just a type and I could have extended QuoteSwap instead of making more cleaners. I think i'll push a fixup